runtime/lua/vim/fs.lua

```diff
@@ -339,13 +339,15 @@ end
 ---@param path (string) Path to normalize
 ---@param opts table|nil Options:
 ---             - expand_env: boolean Expand environment variables (default: true)
+---             - unique_slash: boolean Remove duplicated slashes (default: true)
```

Comment:
```suggestion
---             - expand_env: (boolean) Expand environment variables (default: true)
---             - unique_slash: (boolean) Remove duplicated slashes (default: true)
```

Reply:
Thanks. The `expand_env` line is not a line I added but can I edit this line in my PR (or is it not a big deal)?

Reply:
Fixed in [`0a3876a` (#27202)](https://github.com/neovim/neovim/pull/27202/commits/0a3876aa9852671900bff35f0a35dcbc5db43a38)